mjpc/planners/cem/planner.h

```diff
@@ -0,0 +1,168 @@
+// Copyright 2022 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MJPC_PLANNERS_CEM_OPTIMIZER_H_
+#define MJPC_PLANNERS_CEM_OPTIMIZER_H_
+
+#include <mujoco/mujoco.h>
+
+#include <atomic>
+#include <shared_mutex>
+#include <vector>
+
+#include "mjpc/planners/sampling/planner.h"
+#include "mjpc/planners/planner.h"
+#include "mjpc/planners/cem/policy.h"
+#include "mjpc/states/state.h"
+#include "mjpc/trajectory.h"
+
+namespace mjpc {
+
+// sampling planner limits
```

Comment:
if i don't comment these out, it complains about the fact that these are also defined in `mjpc/planners/sampling/planner.h`. So, I comment them out and import the other file.